web/package.json

```diff
@@ -52,7 +52,7 @@
     "test:ci-report": "CI=true yarn test --coverage --json --outputFile=./test-results/unit-tests/results.json",
     "test:ci": "CI=true yarn test",
     "test:e2e": "sh ./scripts/e2e.sh",
-    "test:e2e-jest": "yarn jest --config ./e2e/jest.config.js ./e2e/test.ts --json --outputFile=./test-results/e2e-tests/results.json",
+    "test:e2e-jest": "yarn jest --config ./e2e/jest.config.js ./e2e --json --outputFile=./test-results/e2e-tests/results.json",
```

Comment:
This line is redundant because we are running **e2e** as bash script, `yarn workspace web test:e2e` it's doing the job